cmake/FMI4cppConfig.cmake.in

```diff
@@ -0,0 +1,17 @@
+get_filename_component(FMI4cpp_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+include(CMakeFindDependencyMacro)
+
+list(APPEND CMAKE_MODULE_PATH ${FMI4cpp_CMAKE_DIR})
+
+find_package(Boost 1.55 REQUIRED)
```

Comment:
I'd recommend using `find_dependency()`[1] here -- that's presumably why you included `CMakeFindDependencyMacro` :)

[1]https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html

Reply:
Thanks, will appy!